### DISCUSSION TITLE
Hey Lars!

### DISCUSSION THREAD

**Discussion by LUISMAGAIVER341** [glzr-io/glazewm#387] [Q&A]
I'm Luís form Brazil.
I'm trying to develop an polybar like app in electron js, to be an YASB alternative.
The main idea, is develop it, for using WITH the Glaze, all cuz the bar isn't nothing much different.
So the idea is develop it using js,html and css to be more personalizable and usable, with a friendly UI and totaly configurable.
Can i get your help?

**Comment by HolbyFPV** [comment]
Sounds like a cool project!

You'll probably want to checkout the [IPC library](https://github.com/glazerdesktop/GlazeWM-js) you could use to communicate between your app and GWM.

Feel free to join our [discord](https://discord.gg/ud6z3qjRvM).

**Comment by LUISMAGAIVER341** [comment]
Very nice!
Unfortunately, i don't have discord.
And I'm not trying to do anything that uses it bar.
And, again, do i have your help?

**Comment by HolbyFPV** [comment]
That's okay.

Feel free to post your problems here on GitHub and we will help if we can.
